dashboard/app/helpers/levels_helper.rb

```diff
@@ -384,13 +384,14 @@ def render_app_dependencies
   end
 
   # As we migrate labs from CDO to Google Blockly, there are multiple ways to determine which version a lab uses:
-  #  1. Setting the useGoogleBlockly view_option, usually configured by a URL parameter.
+  #  1. Setting the blocklyVersion view_option, usually configured by a URL parameter.
   #  2. The corresponding inherited Level model can override Level#uses_google_blockly?. This option is for labs that
   #     have fully transitioned to Google Blockly.
   #  3. The disable_google_blockly DCDO flag, which contains an array of strings corresponding to model class names.
   #     This option will override #2 as an "emergency switch" to go back to CDO Blockly.
   def use_google_blockly
-    return true if view_options[:useGoogleBlockly]
+    return true if view_options[:blocklyVersion] == 'Google'
+    return false if view_options[:blocklyVersion] == 'Cdo'
```

Comment:
i know it's unlikely that we'd add another blocklyVersion, but this might break if we did. is there another way to express this same idea? maybe something like

```ruby
if view_options[:blocklyVersion].present?
  return view_options[:blocklyVersion] == 'Google'
end
```

Reply:
I think a bunch of things would need to change if we added a third blockly version (for example, here https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/_apps_dependencies.html.haml#L51) so I don't think changing just this one place would be of much value.